### DISCUSSION TITLE
Is there a way to search by URL?

### DISCUSSION THREAD

**Discussion by FriedGenera** [sienori/Tab-Session-Manager#1129] [Q&A]
I noticed that the search button only allows to search thru tab/session names, is there an option to allow it to search by url?

**Comment by kissingfrogs2003** [comment]
I tried testing it a few ways but I don't think this feature exists. Just the tab/session names from what I can see.

It might be a good future enhancement idea though not sure it will be a top priority one. Sorry :(

**Reply by Blook4** [reply]
very needed
